### PR TITLE
Added Twitter and Reddit base pixels and checkout_confirmation.html to Gluteboost container

### PR DESCRIPTION
**Pull request by ankitnagarsheth**
Added Twitter and Reddit base pixels to Gluteboost container for merging with Base container after reviewing it.

And, also added checkout_confirmation.html for gluteboost

### CODE REVIEW

File: gtm/GTM Container for Gluteboost.json [hunk 1]
```diff
@@ -0,0 +1,7894 @@
...
+                "formatValue": {}
+            },
+            {
+                "accountId": "6073593102",
+                "containerId": "102923929",
+                "variableId": "91",
+                "name": "JS - Ecommerce - Add - Items - Meta",
+                "type": "jsm",
+                "parameter": [
+
```
Comment (pazwicker) [inline gtm/GTM Container for Gluteboost.json:7663]: @ankitnagarsheth the merchant has two Facebook pixels? Is this valid?

**Review by pazwicker — CHANGES_REQUESTED** [review]
@ankitnagarsheth please also update the baseline GTM container with Gluteboost's version. Please just change all the custom variables' values to `[UPDATE]`.

File: gtm/GTM Container for Gluteboost.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ankitnagarsheth) [reply gtm/GTM Container for Gluteboost.json:6825]: that is correct. They do not use it.

Comment (ankitnagarsheth) [reply gtm/GTM Container for Gluteboost.json:7663]: Thats right! They have 2 facebook pixels.

File: gtm/GTM Container for Gluteboost.json [hunk 2]
```diff
@@ -0,0 +1,7894 @@
...
+                        "parameter": [
+                            {
+                                "type": "TEMPLATE",
+                                "key": "arg0",
+                                "value": "{{Constant - CookieBot T/F}}"
+                            },
+                            {
+                                "type": "TEMPLATE",
+                                "key": "arg1",
+                                "value": "false"
```
[the remark this answers is not in the document]
Comment (ankitnagarsheth) [reply gtm/GTM Container for Gluteboost.json:3843]: This has been changed to true.

**Comment by ankitnagarsheth** [comment]
https://github.com/pazwicker/nyla-analytics/pull/16#pullrequestreview-1404466426
@pazwicker  I will be updating the baseline container tomorrow EOD.

**Comment by ankitnagarsheth** [comment]
updated the baseline GTM container with Gluteboost's version. And updated custom variables' values to [UPDATE].

**Review by pazwicker — CHANGES_REQUESTED** [review]

File: merchants/gluteboost/theme.liquid
[the remark this answers is not in the document]
Comment (ankitnagarsheth) [reply merchants/gluteboost/theme.liquid:1]: @pazwicker  The theme file was a local copy. But, it is not needed in main branch. So, I have removed it.

File: gtm/baseline_nyla_gtm_gluteboost.json
[the remark this answers is not in the document]
Comment (ankitnagarsheth) [reply gtm/baseline_nyla_gtm_gluteboost.json:1]: @pazwicker the merchant-specific IDs are removed and it is now referencing [UPDATE]. The filename is updated and references are removed to Gluteboost.

**Review by pazwicker — APPROVED** [review]
looks good!

**Pull request merged**